src/vcpkg/commands.create.cpp

```diff
@@ -45,13 +45,13 @@ namespace vcpkg::Commands::Create
 
         if (parsed.command_arguments.size() >= 3)
         {
-            const std::string& zip_file_name = parsed.command_arguments[2];
+            std::string& zip_file_name = parsed.command_arguments[2];
             Checks::msg_check_exit(VCPKG_LINE_INFO,
                                    !has_invalid_chars_for_filesystem(zip_file_name),
                                    msgInvalidFilename,
                                    msg::value = FILESYSTEM_INVALID_CHARACTERS,
                                    msg::path = zip_file_name);
-            cmake_args.emplace_back("FILENAME", zip_file_name);
+            cmake_args.emplace_back("FILENAME", std::move(zip_file_name));
```

Comment:
I'm very nervous about destroying command_arguments by proxy like this; I don't think saving this 1 parameter copy is worth that.